.github/workflows/package-tests.yml

```diff
@@ -100,4 +100,41 @@ jobs:
               fi
           done
 
+          exit $ok
+      - name: Verify symfony/deprecation-contracts requirements
+        run: |
+          ok=0
+
+          json='${{ steps.find-packages.outputs.packages }}'
+          for package in $(echo "${json}" | jq -r '.[] | @base64'); do
+              _jq() {
+                echo ${package} | base64 --decode | jq -r ${1}
+              }
+
+              NAME=$(_jq '.name')
+              if [[ $NAME = 'symfony/deprecation-contracts' || $NAME = 'symfony/contracts' ]]; then
+                continue
+              fi
+
+              echo "::group::$NAME"
+              DIR=$(_jq '.directory')
+              localExit=0
+              grep -rq 'trigger_deprecation(' --include=*.php --exclude-dir=Tests/ --exclude-dir=Bridge/ $DIR
+              triggersDeprecation=$?
+              if [[ $triggersDeprecation -eq 0 && $(_jq '.requires_deprecation_contracts') == false ]]; then
```

Comment:
if there is at least one trigger_deprecation( match and the dep is not required -> fail